src/services/rhsm/rhsmConstants.js

```diff
@@ -6,13 +6,15 @@
 /**
  * RHSM path IDs for product RHEL variants.
  *
- * @type {{RHEL_ARM: string, RHEL_X86_SAP: string, RHEL_IBM_Z: string, RHEL_IBM_POWER: string, RHEL_X86: string}}
+ * @type {{RHEL_ARM: string, RHEL_X86_SAP: string, RHEL_IBM_Z: string, RHEL_IBM_POWER: string, RHEL_X86: string,
+ * RHEL_X86_EUS: string}}
```

Comment:
required: just indent line `10` 4 more spaces and you should be good here

Reply:
indented